### ISSUE TITLE
Talos-data depends prf-ros-controllers even on erbium

### ISSUE THREAD

**Issue by NoelieRamuzat** [Gepetto/robotpkg-wip#1]
Hi ! 
In the Makefile of talos-data there is a dependency to py-prf-gazebo-ros-pkgs:
`include ../../wip/py-prf-gazebo-ros-pkgs/depend.mk`

But in erbium (like on Talos) it is not necessary (already installed). 
We should do like in talos-metapkg-ros-control-sot:

```
ifeq (erbium,${PREFER.pal-distro})
  ifeq (palerbium,${PALDISTRO})
    CMAKE_PREFIX_PATH+=/opt/pal/erbium
  else
    include ../../wip/py-prf-ros-controllers/depend.mk
  endif

displaytalosmetapkg:
	echo 'PREFER.pal-distro:${PREFER.pal-distro}'
	echo 'pal-distro-erbium'

else
  include ../../wip/py-prf-ros-controllers/depend.mk

displaytalosmetapkg:
	echo 'PREFER.pal-distro:${PREFER.pal-distro}'
	echo 'pal-distro-xenial'

endif
```

Thanks !

**Comment by nim65s** [comment]
talos-data depends on prf-ros-controllers, this is just a fact.

But on erbium, robotpkg should find prf-ros-conrtollers on /opt/pal/erbium and correctly setup the required environment variable for it, instead of trying to build it. I think your issue is there: 
https://github.com/Gepetto/robotpkg-wip/blob/0aeb6db7f1d34440f6452ae75ace8a6e66fd1840/py-prf-gazebo-ros-pkgs/depend.mk#L17

On erbium, this `PREFER` should be `system`.

Please try to add `PREFER.py-prf-gazebo-ros-pkgs = system` in your `robotpkg.conf` to check that.

**Comment by NoelieRamuzat** [comment]
I tried but I have the same error:
```
Missing system packages required for talos-data-1.2.2:
py27-prf-gazebo-ros-pkgs>=3.0.1
```

**Comment by nim65s** [comment]
Fixed in 73e5242b3d18d31f4b4cd9d282fb729fb6cc52ba & 3ab7f7c31805abd33a4120260515c37b26b7ac15.

But there are probably other issues, I'll try to get `py-talos-dev` work from source on nakasan.
